apps/test/integration/levelSolutions/applab1/ec_canvas.js

```diff
@@ -143,86 +141,6 @@ module.exports = {
         result: true,
         testResult: TestResults.FREE_PLAY
       }
```

Comment:
I had trouble getting this level test working in both PhantomJS and Chrome.  It's testing a polyfill behavior for the `movementX` and `movementY` properties on a mouse event, so it assumes we're generating a synthetic mouse event that is missing those properties - this is true by default in Phantom, so the test passes on Drone today.

Unfortunately, Chrome synthetic mouse events not only include `movementX` and `movementY` already, but they are readonly properties that cannot be removed. They're also not easy to initialize for a synthetic event. That causes this test to fail, because EventSandboxer tries to use the real movementX/Y values on the synthetic event, but they're zero instead of accurate values for the movement the test is simulating.

I could see us taking a couple of different approaches to fix this test:

1. We could update our `createMouseEvent` utility to generate a _less_ realistic mouse event so that Chrome can test the fallback case as expected. The downside of this is that it might be a large chunk of work and seems like the sort of thing that would cause headaches in the future.
2. We could update the test to try and provide realistic movementX/Y properties.  The downside of this is that the test is no longer really checking the polyfill behavior it cares about.  It's also not really supported to set these properties on a synthetic event so this may not work on Phantom.

Instead, I am removing this test. It was written by Dave in 2015 when the computed movementX/Y feature was
introduced: #6138

However, a few months later, this behavior was extracted out to the EventSandboxer class and thoroughly unit-tested there: #6752

So I propose coverage of this feature does not get much worse by pulling this test, and we'll have one less level solution test (notoriously difficult to debug) to maintain in the future.